jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java

```diff
@@ -177,9 +177,14 @@ private List<Map<String, Object>> fetchEntitiyInfoFromS3(String datasource, Stri
         LOGGER.debug("s3Role:{}", s3Role);
         LOGGER.debug("bucketName:{}", bucketName);
         LOGGER.debug("sourceFolder:{}", sourceFolder);
+        LOGGER.debug("datasource:{}", datasource);
         try {
-            //override datapath with sourceFolder
-            dataPath = sourceFolder;
+            //override datapath with sourceFolder(i.e) for redhat s3.data will come from Cq-mapper
+            if (datasource.equalsIgnoreCase("redhat")) {
```

Comment:
We have this logic for redhat when the data will come in s3.data based on Arun's comments